spec/factories/omniauth.rb

```diff
@@ -0,0 +1,37 @@
+FactoryGirl.define do
+
+  factory :auth_hash, class: OmniAuth::AuthHash do
+    skip_create
+
+    transient do
+      sequence(:uid)
+      provider "provider"
+      email { "email" }
+    end
```

Comment:
That makes sense.

I was trying to use unique credentials for each test that uses this hash and for some reason(probably saw an example somewhere) used `transient` for `sequence(:uid)` since I couldn't use that in my hash. It works fine in my traits so I removed the `transient` section.

Thanks @obulkin